docs/src/pages/components/dialogs/ConfirmationDialog.js

```diff
@@ -61,8 +61,6 @@ function ConfirmationDialogRaw(props) {
 
   return (
     <Dialog
-      disableBackdropClick
-      disableEscapeKeyDown
```

Comment:
They don't have any effect on `next` either because we already have a fully custom `onClose` in `ConfirmationDialogRaw`